packages/web-components/src/components/cbp-tooltip/cbp-tooltip.tsx

```diff
@@ -59,10 +70,10 @@ export class CbpTooltip {
             type="button"
             fill="ghost"
             color="secondary" 
-            context={(this.context == 'dark-always' || this.context == 'dark-inverts')? "light-always":"dark-always"}
+            context={(document.getElementsByTagName("cbp-app")[0].theme == 'dark' || this.context == 'dark-always' || this.context == 'dark-inverts')? "light-always":"dark-always"}
```

Comment:
I've never had to do this sort of logic before. I think `dark-inverts` should cover it. This represents the default state (it's a dark context, so the button will appear in dark mode when the user is in light mode) but it inverts (to light) when the user switches to dark mode.